src/styles/font-jrkf/iconfont.js

```
@@ -0,0 +1,68 @@
+            (l.style.overflow = 'hidden'),
+            (l = l),
+            (a = document.body).firstChild
+              ? n(l, a.firstChild)
+              : a.appendChild(l))
+      }),
+        document.addEventListener
+          ? ~['complete', 'loaded', 'interactive'].indexOf(document.readyState)
+            ? setTimeout(h, 0)
+            : ((t = function () {
+                document.removeEventListener('DOMContentLoaded', t, !1), h()
+              }),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)